internal/workflow_shadower_test.go

```diff
@@ -203,7 +203,7 @@ func (s *workflowShadowerSuite) TestShadowOptionsValidation() {
 			options:   ShadowOptions{},
 			expectErr: false,
 			validationFn: func(options *ShadowOptions) {
-				s.Equal("(CloseTime = missing)", options.WorkflowQuery)
+				s.Equal("(CloseTime = missing or CloseTime != missing)", options.WorkflowQuery)
```

Comment:
is not this equivalent to empty query?

Reply:
it is but the query builder forms it this way because of open and closed statuses getting added together in the test. This portion of default status check needs a rewrite that I will be doing as a part of the shadower fix initiative.

Reply:
we do have an ALL status filter somewhere but it barely gets used. Now the problem with that one is that it picks up everything present in that defaut list of statuses.